arrow/api.py

```diff
@@ -8,19 +8,23 @@
 from __future__ import absolute_import
 
 from arrow.factory import ArrowFactory
+from functools import wraps
 
 
 # internal default factory.
 _factory = ArrowFactory()
 
 
+@wraps(_factory.get)
```

Comment:
`@wraps` does more than just insert the documentation.  I'd prefer to see `get.__doc__ = _factory.get.__doc__`, to ensure that only the docstring is modified.

Reply:
OK Agreed -- done